tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -486,6 +486,8 @@ export const isMetricsSlideoutMenuOpen = createSelector(
 export const getMetricsCardMinMax = createSelector(
   getCardStateMap,
   (cardStateMap: CardStateMap, cardId: CardId): MinMaxStep | undefined => {
+    if (!Object.keys(cardStateMap).includes(cardId)) return undefined;
```

Comment:
nit: `Array.includes` has a runtime of O(N) while something like this has a runtime of O(1). Also would you mind adding/updating a test for this?
```suggestion
    if (!cardStateMap[cardId]) return;
```